patches/chrome-renderer-worker_content_settings_client.cc.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/chrome/renderer/worker_content_settings_client.cc b/chrome/renderer/worker_content_settings_client.cc
+index 99502cf0d5e3ab805be3f2d0ff2590ec41dd546d..e0a5b7ae9c4f5ca22edf3d62a23742ecd67a035d 100644
+--- a/chrome/renderer/worker_content_settings_client.cc
++++ b/chrome/renderer/worker_content_settings_client.cc
+@@ -43,6 +43,7 @@ WorkerContentSettingsClient::WorkerContentSettingsClient(
+     content_setting_rules_ =
+         std::make_unique<RendererContentSettingRules>(*rules);
+   }
++  BRAVE_WORKER_CONTENT_SETTINGS_CLIENT
```

Comment:
why are we patching this instead of subclassing?

Reply:
solved with a subclass, thanks.